src/lib/renderComponent.tsx

```diff
@@ -118,6 +119,21 @@ const renderWithFocusZone = (render, focusZoneDefinition, config, focusZoneRef):
   return render(config)
 }
 
+const getProviderNotUsedWarningElement = (displayName: string) => {
+  console.error(`Incorrect usage. Your component needs to be wrapped using the Provider component:
+
+import * as React from 'react';
+import { ${displayName}, Provider, themes } from '@stardust-ui/react';
+
+<Provider theme={themes.teams}>
+  <${displayName} />
+</Provider>
+`)
```

Comment:
Suggest telling the user that they "Did you place a Provider at the top of your app?"

Reply:
done